app/src/lib/stores/accounts-store.ts

```diff
@@ -159,7 +155,9 @@ export class AccountsStore extends TypedBaseStore<ReadonlyArray<Account>> {
       return
     }
 
-    this.accounts = this.accounts.filter(a => a.id !== account.id)
+    this.accounts = this.accounts.filter(
+      a => !(a.endpoint === account.endpoint && a.id === account.id)
```

Comment:
`.id` here is the endpoint-assigned id, not a local database id. While it's extremely (unless you're user id #1 on GitHub.com) unlikely that there'd ever be a collision here it's technically scoped by endpoint and therefore we should compare it against endpoint as well.

Reply:
✨Ah, good point! Nice improvement ✨